rpg/Global/GlobalCommands.cs

```diff
@@ -39,6 +49,75 @@ public void GetStatistics(Client sender)
                     // TODO: Cop ranks, experience
                 }
             }
+        }
+
+
+        [Command("me")]
+        public void emotes(Client sender, string text)
+        {
+            string name = "~p~" + sender.name +" " + text;
+            API.sendChatMessageToAll(name);
+
+        }
+
+        [Command("pm")]
+        public void message(Client sender, Client reciever, string text)
+        {
+            API.sendChatMessageToPlayer(reciever, "~r~"+ sender.name + " tells you ", text);
+
+        }
+
+        [Command("shout")]
+        public void shout(Client sender, string text)
+        {
+
+            API.sendChatMessageToAll("~h~~b~" + sender.name + " shouts " + text);
+
+        }
+
+        [Command("give")]
+        public void givemoney (Client sender,Client player,int amount) 
+         {
+
+            if (API.getEntityData(sender, "Money") < amount)

```

Comment:
How does this work? This only transfers money if the sender does *not* have enough money!?

This function also doesn't *take* money from the sender. This is completely broken!